telegram/_keyboardbutton.py

```diff
@@ -58,10 +58,10 @@ class KeyboardButton(TelegramObject):
             sent as a contact when the button is pressed. Available in private chats only.
         request_location (:obj:`bool`, optional): If :obj:`True`, the user's current location will
             be sent when the button is pressed. Available in private chats only.
-        request_poll (:class:`KeyboardButtonPollType`, optional): If specified, the user will be
-            asked to create a poll and send it to the bot when the button is pressed. Available in
-            private chats only.
-        web_app (:class:`WebAppInfo`, optional): If specified, the described `Web App
+        request_poll (:class:`~telegram.KeyboardButtonPollType`, optional): If specified, the user
```

Comment:
is this necessary for the automatic detection to work? If so: I dare say that we'll at some point forget about this and it would be nice if the code could handle it 😬 I guess `getattr(telegram, "KeyboardButtonPollType")` should work in this case?

Reply:
I changed the procedure slightly to make sure the "telegram" word is not needed. I'll probably leave the docstrings as I changed them, though, because this way they're consistent with args.